### PR TITLE
chore: remove `REMIX_TOKEN` references

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

File: other/docker-build [hunk 1]
```diff
@@ -1 +1 @@
-docker build . --build-arg REMIX_TOKEN=${REMIX_TOKEN}
\ No newline at end of file
+docker build .
```
Comment (kentcdodds) [inline other/docker-build]: Wonder if we can get rid of the docker build script file

**Review by kentcdodds — COMMENTED** [review]
Do you know if that file was used anywhere?

**Comment by MichaelDeBoey** [comment]
@kentcdodds I checked for code references, but couldn't find any usage

**Comment by kentcdodds** [comment]
Thanks!

**Comment by MichaelDeBoey** [comment]
@kentcdodds Build & deploy seems to work 👍
https://github.com/kentcdodds/kentcdodds.com/runs/4637430239

**Pull request merged**